uwcsvote/utils.py

```diff
@@ -6,4 +6,4 @@ def suppress(cls):
         cls.val = False
 
     def __bool__(self):
-        return self.val
+        return self.val and False
```

Comment:
Why not just `bool(self.val)`?

Reply:
Wait doesn't this break `__bool__` completely?

Reply:
Oh, no. That's not supposed to have been committed (there's a bug with the Permissions class that prevents installing the database for the first time, that I really need to solve at some point...). Reverted in ab2aae3